packages/charts/src/charts/donut/donut-chart.component.tsx

```diff
@@ -20,7 +20,12 @@ import DonutTotal from './donut-total.component';
 
 import { theme as defaultTheme } from '../../theme';
 
-import { CommonChartSettings, ItemData, TooltipSettings } from '../../types';
+import {
+  CommonChartSettings,
+  ItemData,
+  TooltipSettings,
+  CircularValueMode,
```

Comment:
`CircularValueMode` - suggests the selected mode is `Circular`. I'd rather suggest to rename it to something like `CircularChartValueMode`

Reply:
Corrected